infrastructure/ansible/roles/traffic_ops/templates/influxdb.conf.j2

```diff
@@ -0,0 +1,20 @@
+{#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#}
+{
+    "cache_stats_db_name": "{{ to_influxdb_cache_stats_db_name  }}",
+    "delivery_stats_db_name": "{{ to_influxdb_deliveryservice_stats_db_name }}",
+    "password": "{{ to_influxdb_password }}",
+    "user": "{{ to_influxdb_user }}",
+    "secure": {{ to_influxdb_secure | lower }}
```

Comment:
The `to_influxdb_secure` variable should be treated as a boolean (and enforced to run through a boolean filter) before running through lower to get true || false